content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -65,7 +65,7 @@ The `.spec.schedule` field is required. The value of that field follows the [Cro
 # * * * * *
 ```
 
-For example, `0 0 13 * 5` states that the task must be started every Friday at midnight, as well as on the 13th of each month at midnight.
+For example, `0 0 1,10,20 * 1,3,5` means this cron job is scheduled to run at midnight on the 1st, 10th, and 20th of each month, but only if those days fall on a Monday, Wednesday, or Friday.
```

Comment:
This is a complex crontab too -- requiring both a specific set of dates and days of the week.

Could we use something that triggers weekly on a Monday at 3am? I don't think this example needs to be super in depth, folks can go to the cron docs for complex cron setup.